docs/expandable/elements.md

```diff
@@ -0,0 +1,49 @@
+## Import
+
+```js
+
+```
```

Comment:
I don't think we need this section in elements components. It wasn't there in [Fabric Elements](https://elements.fabric-ds.io/expandable.html) either.